code/game/machinery/cloning.dm

```diff
@@ -358,14 +358,13 @@
 		mob_occupant.grab_ghost()
 		to_chat(occupant, "<span class='notice'><b>There is a bright flash!</b><br><i>You feel like a new being.</i></span>")
 		mob_occupant.flash_act()
-
 	occupant.forceMove(T)
 	icon_state = "pod_0"
 	mob_occupant.domutcheck(1) //Waiting until they're out before possible monkeyizing. The 1 argument forces powers to manifest.
 	for(var/fl in unattached_flesh)
 		qdel(fl)
 	unattached_flesh.Cut()
-
+	mob_occupant.setCloneLoss(0)

```

Comment:
if you want it removed at least do it properly by refactoring the cloning process to no longer rely on cloneloss

Reply:
read the GICJING DESVROPTOOM

Reply:
That is irrelevant, you need to untie the counting process from cloneloss to do this properly

Reply:
OH MY GOD YOU RETARDED FUCK

CLONELOSS IS STILL USED BY BLOBS FOR WHATEVER REASON AND ADDITIONALLY WHY WOULD I DEFINE ANOTHER FUCKING TICKER VAR WHEN CLONELOSSS WORKS JUST FINE HOLY SHIT READ THE FUCKING DESCRIPTION

Reply:
Because this is a shitty hack

Reply:
FUCK YOU